### PR TITLE
Add config for setting latest tag

### PR DESCRIPTION
**Pull request by danield9tqh**
## Summary
Docker latest tag is used as the default tag when pulling a docker image and we only want to do this during a release, not if we are deploying specific versions.

## Testing Plan
Tested deploy locally + in github actions

## Documentation

Does this change require any updates to the Iron Fish Docs (ex. [the RPC API
Reference](https://ironfish.network/docs/onboarding/rpc/chain))? If yes, link a
related documentation pull request for the website.

```
[ ] Yes
```

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: .github/workflows/deploy-node-aws.yml [hunk 1]
```diff
@@ -1,5 +1,11 @@
 name: Deploy Node Image to AWS
-on: workflow_dispatch
+on:
+  workflow_dispatch:
+    inputs:
+      tag_latest:
+        description: 'Tag this docker image as lastest. Should only be used for production releases'
```
Comment (hughy) [inline .github/workflows/deploy-node-aws.yml:6]:
```suggestion
        description: 'Tag this docker image as latest. Should only be used for production releases'
```

File: .github/workflows/deploy-node-github-beta.yml [hunk 2]
```diff
@@ -1,5 +1,12 @@
 name: Deploy Node Image to Github Beta
-on: workflow_dispatch
+on:
+  workflow_dispatch:
+    inputs:
+      tag_latest:
+        description: 'Tag this docker image as lastest. Should only be used for production releases'
```
Comment (hughy) [inline .github/workflows/deploy-node-github-beta.yml:6]:
```suggestion
        description: 'Tag this docker image as latest. Should only be used for production releases'
```

File: .github/workflows/deploy-node-github.yml [hunk 3]
```diff
@@ -1,5 +1,11 @@
 name: Deploy Node Image to Github
-on: workflow_dispatch
+on:
+  workflow_dispatch:
+    inputs:
+      tag_latest:
+        description: 'Tag this docker image as lastest. Should only be used for production releases'
```
Comment (hughy) [inline .github/workflows/deploy-node-github.yml:6]:
```suggestion
        description: 'Tag this docker image as latest. Should only be used for production releases'
```

**Review by hughy — APPROVED** [review]

**Comment by danield9tqh** [comment]
Closing this in favor of https://github.com/iron-fish/ironfish/pull/3701

**Pull request closed without merging**